webapp/apps/test_assets/test_param_formatters.py

```diff
@@ -3,7 +3,10 @@
 import taxcalc
 import numpy as np
 
-from ..taxbrain.views import read_json_reform, parse_errors_warnings
+from ..taxbrain.views import (read_json_reform, parse_errors_warnings,
```

Comment:
One handy software engineering tool you might be interested in is "blame". Basically, it's a tool that shows the last person to make changes to each line of code. GitHub has a more thorough explanation [here](https://help.github.com/articles/tracing-changes-in-a-file/). This isn't a review comment per se, but one thing to keep in mind when using that tool is writing code in a way that "preserves blame", in other words, so that you don't make changes to lines that you don't edit. In this case, the best way to "preserve blame" would be to write each import on a new line, so that if you add your own import, it doesn't update the blame on someone else's import, so you know the last person who thought that import was important to change. i.e.

```python
from ..taxbrain.views import (
   read_json_reform,
   parse_errors_warnings,
   ...
)
```

Reply:
I hadn't thought about preserving "blame" before. That makes a lot of sense. Thanks for the pointer.

Reply:
Check out the blame tool, it is useful for figuring out who wrote something if you're trying to figure out why it was done that way (i.e. poorly documented code)